### DISCUSSION TITLE
Using only one service for backup and database?

### DISCUSSION THREAD

**Discussion by thibaut-d** [prodrigestivill/docker-postgres-backup-local#113] [Q&A] [answered]
Hi,

I have seen that the Docker image is based on Postgres image.

Would it be possible to use this image directly as a database service and thus avoiding using 2 services on low RAM systems?

I don't know if this relevant however.

Regards

**Comment by dargmuesli** [comment]
I do not see anything that speaks against it.

**Comment by dargmuesli** [comment]
The only potential issues that came to my mind are tags, as could differ from the original Postgres image's tags and updated versioning in general: is automatic rebuilding of this image when the base image changes enabled?

**Comment by prodrigestivill** [comment, marked as answer]
> Would it be possible to use this image directly as a database service and thus avoiding using 2 services on low RAM systems?

You can use this image to run the database, but you will not save much RAM nor disk space.
Since the actual programs running will be the same. And both images share almost all files so docker saves disk space.

> is automatic rebuilding of this image when the base image changes enabled?

Yes, Docker Hub auto-builds are enabled, so when PostgreSQL image are updated this ones should be rebuild.
